docs/tutorials/upgrade-extension.md

```diff
@@ -11,34 +11,310 @@ For information on downgrading an extension, see [Downgrade an Extension](downgr
+      verbs: [get, list, watch]
+    - apiGroups: [monitoring.coreos.com]
+      resources: [prometheuses]
+      verbs: ['*']
+    - apiGroups: [monitoring.coreos.com]
+      resources: [servicemonitors]
+      verbs: ['*']
+    - apiGroups: [networking.k8s.io]
+      resources: [ingresses]
+      verbs: ['*']
+    - apiGroups: [rbac.authorization.k8s.io]
+      resources: ['*']
+      verbs: ['*']
+    - apiGroups: [rbac.authorization.k8s.io]
+      resources: [clusterrolebindings]
+      verbs: ['*']
+    - apiGroups: [rbac.authorization.k8s.io]
+      resources: [clusterroles]
+      verbs: ['*']
+    - apiGroups: [route.openshift.io]
+      resources: [routes]
+      verbs: ['*']
+    - apiGroups: [route.openshift.io]
+      resources: [routes/custom-host]
+      verbs: ['*']
+    - apiGroups: ["coordination.k8s.io"]
+      resources: ["leases"]
+      verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+    ---
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: Role
+    metadata:
+      name: argocd-installer-role
+      namespace: argocd
+    rules:
+    - apiGroups: [""]
+      resources: [serviceaccounts]
+      verbs: [create, list, watch]
+    - apiGroups: [""]
+      resources: [serviceaccounts]
+      verbs: [get, update, patch, delete]
+      resourceNames: [argocd-operator-controller-manager]
+    - apiGroups: [""]
+      resources: [configmaps]
+      verbs: [create, list, watch]
+    - apiGroups: [""]
+      resources: [configmaps]
+      verbs: [get, update, patch, delete]
+      resourceNames: [argocd-operator-manager-config]
+    - apiGroups: [""]
+      resources: [services]
+      verbs: [create, list, watch]
+    - apiGroups: [""]
+      resources: [services]
+      verbs: [get, update, patch, delete]
+      resourceNames: [argocd-operator-controller-manager-metrics-service]
+    - apiGroups: [apps]
+      resources: [deployments]
+      verbs: [create, list, watch]
+    - apiGroups: [apps]
+      resources: [deployments]
+      verbs: [get, update, patch, delete]
+      resourceNames: [argocd-operator-controller-manager]
+    ---
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: RoleBinding
+    metadata:
+      name: argocd-installer-binding
+      namespace: argocd
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: Role
+      name: argocd-installer-role
+    subjects:
+    - kind: ServiceAccount
+      name: argocd-installer
+      namespace: argocd
+    ---
+    apiVersion: olm.operatorframework.io/v1
+    kind: ClusterExtension
+    metadata:
+      name: argocd
+    spec:
+      namespace: argocd
+      serviceAccount:
+        name: argocd-installer
+      source:
+        sourceType: Catalog
+        catalog:
+          packageName: argocd-operator
+          version: 0.2.0
+    ```
+
+If we view the current state of our ClusterExtension we should see that we have installed version 0.2.0:
+
+```terminal
+kubectl get clusterextension argocd -o yaml | tail -n4
```

Comment:
Maybe worth using `jsonpath` or `jsonpath-as-json` instead of `tail` here? Or just show full yaml?

```
# kubectl get clusterextension argocd -o jsonpath="{.status.install}" | yq -p json
bundle:
  name: argocd-operator.v0.2.0
  version: 0.2.0
```

```
# kubectl get clusterextension argocd -o jsonpath-as-json="{.status.install}"
[
    {
        "bundle": {
            "name": "argocd-operator.v0.2.0",
            "version": "0.2.0"
        }
    }
]
```

Reply:
Just to clarify: I'm suggesting this because I'm not sure if field ordering is guaranteed somehow in `kubectl get -o yaml` output.

Reply:
I think it would clean up this whole doc quite a bit to just use this jsonpath-as-path command for both instances where we check the clusterextension status. This would also allow us to remove that last note I added about the last-applied-configuration annotation since that won't show up in the tutorial anymore. What do you think?

Reply:
I guess the note can stay, in case anyone does notice that discrepancy. Doesn't hurt anything.

Reply:
I had a look at the updated doc - looks good to me. Resolving this thread.